core/src/main/java/com/adobe/granite/translation/connector/bootstrap/core/impl/BootstrapTranslationServiceFactoryImpl.java

```diff
@@ -84,26 +93,41 @@ public TranslationService createTranslationService(TranslationMethod translation
 			throws TranslationException {
 		log.trace("BootstrapTranslationServiceFactoryImpl.createTranslationService");
 
-		BootstrapTranslationCloudConfig bootstrapCloudConfg = (BootstrapTranslationCloudConfig) cloudConfigUtil
-				.getCloudConfigObjectFromPath(BootstrapTranslationCloudConfig.class, cloudConfigPath);
 
-		String dummyConfigId = "";
-		String dummyServerUrl = "";
+		BootstrapTranslationCloudConfig bootstrapCloudConfg = null;
+    LiltApiClient liltApiClient = null;
+
+		String liltConfigId = "";
+		String liltServerUrl = "";
 		String previewPath = "";
 
+    try {
+      Map<String, Object> param = new HashMap<String, Object>();
```

Comment:
Minor style thing - you can infer the type parameters in the constructor:

```suggestion
      Map<String, Object> param = new HashMap<>();
```